### PR TITLE
[numpy] add op fabs, sometrue, round_

### PR DESCRIPTION
**Pull request by Yiyan66**
## Description ##
add op fabs, sometrue, round_

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at https://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the best of my knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

File: python/mxnet/_numpy_op_doc.py [hunk 1]
```diff
@@ -121,6 +121,22 @@ def _np_any(a, axis=None, keepdims=False, out=None):
     pass
 
 
+
```
Comment (haojin2) [inline python/mxnet/_numpy_op_doc.py]: get rid of this blank line

File: python/mxnet/ndarray/numpy/_op.py [hunk 2]
```diff
@@ -39,7 +39,7 @@
            'average', 'mean', 'maximum', 'minimum',
            'swapaxes', 'clip', 'argmax', 'argmin', 'std', 'var', 'indices', 'copysign', 'ravel', 'unravel_index',
            'diag_indices_from', 'hanning', 'hamming', 'blackman', 'flip', 'flipud', 'fliplr', 'around', 'round',
-           'hypot', 'bitwise_and', 'bitwise_xor', 'bitwise_or', 'rad2deg', 'deg2rad', 'unique', 'lcm',
+           'round_', 'hypot', 'bitwise_and', 'bitwise_xor', 'bitwise_or', 'rad2deg', 'deg2rad', 'unique', 'lcm',
```
Comment (haojin2) [inline python/mxnet/ndarray/numpy/_op.py]: maybe move this to the position after `around` and `round`, both in this list and in the file. Same for all other files as well.

File: python/mxnet/numpy/multiarray.py [hunk 3]
```diff
@@ -1572,6 +1572,14 @@ def clip(self, min=None, max=None, out=None):  # pylint: disable=arguments-diffe
         """
         return clip(self, min, max, out=out)
 
+    def fabs(self, *args, **kwargs):
```
Comment (haojin2) [inline python/mxnet/numpy/multiarray.py]: no need for adding non-existent convenience method

File: src/operator/numpy/np_elemwise_unary_op_basic.cc [hunk 4]
```diff
@@ -133,6 +133,7 @@ Example::
 // abs
 MXNET_OPERATOR_REGISTER_NUMPY_UNARY(_npi_absolute, "x", mshadow_op::abs)
 .add_alias("_npi_abs")
+.add_alias("_npi_fabs")
```
Comment (haojin2) [inline src/operator/numpy/np_elemwise_unary_op_basic.cc]: seems like you're only using `npi.abs` in your Python wrappers, then no need for this alias.

File: python/mxnet/symbol/numpy/_symbol.py [hunk 5]
```diff
@@ -523,6 +523,14 @@ def abs(self, *args, **kwargs):
         """
         raise AttributeError('_Symbol object has no attribute abs')
 
+    def fabs(self, *args, **kwargs):
```
Comment (haojin2) [inline python/mxnet/symbol/numpy/_symbol.py]: no need for this convenient method either.

File: python/mxnet/_numpy_op_doc.py [hunk 6]
```diff
@@ -121,6 +121,21 @@ def _np_any(a, axis=None, keepdims=False, out=None):
     pass
 
 
+def _np_sometrue(a, axis=None, keepdims=False, out=None):
+    """
+    sometrue(*args, **kwargs)
```
Comment (reminisce) [inline python/mxnet/_numpy_op_doc.py]: No need to have this line.

**Pull request merged**